app/build.gradle.kts

```diff
@@ -26,8 +26,8 @@ android {
         applicationId = "org.eclipse.kuksa.testapp"
```

Comment:
During "./gradlew publishToMavenLocal" I noticed a few warnings about duplicate classes, I think they can be neglected, because they only appear during the creation of the "releaseSourcesJar"

Reply:
I don't see duplicates inside the jar, so fine for me.